WaveToyX/test/presync.par

```diff
@@ -35,7 +36,6 @@ CarpetX::out_norm_omit_sumloc_for_backward_compatibility = yes
 
 CarpetX::out_tsv_vars = "
     WaveToyX::state
-    WaveToyX::rhs
```

Comment:
This change seems unrelated – why is the test weakened?

Reply:
good question, no idea why. I'll put it back.

Reply:
updated, is not a test so nothing failed

Reply:
ah, the output file (`WaveToyX/test/presync/wavetoyx-rhs.it000000.x.tsv` or so) was never committed to git, so there was no actual test. See 89f1a58d